src/ResourceManager/DataFactories/Commands.DataFactories/Commands.DataFactories.Netcore.csproj

```diff
@@ -18,6 +18,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <DelaySign>false</DelaySign>
     <DefineConstants>TRACE;DEBUG;NETSTANDARD</DefineConstants>
+    <NoWarn>1701;1702;NU1701</NoWarn>
```

Comment:
We don't allow excluding warnings in Az. All warnings are treated as errors and should be resolved accordingly. Otherwise, one module can unintentionally break another module.

Reply:
Removed.